x-pack/platform/plugins/shared/streams/server/lib/streams/component_templates/generate_layer.ts

```diff
@@ -63,12 +66,54 @@ export function generateLayer(
     version: ASSET_VERSION,
     _meta: {
       managed: true,
-      description: `Default settings for the ${name} stream`,
+      description: `Default settings for the ${definition.name} stream`,
     },
   };
 }
 
-function getTemplateLifecycle(definition: WiredStreamDefinition, isServerless: boolean) {
+export function generateUnwiredBaseLayer(
+  definition: UnwiredStreamDefinition,
+  existingTemplate: IndicesIndexTemplate
+): ClusterPutComponentTemplateRequest {
+  return {
+    name: getBaseLayerComponentName(definition.name),
+    template: {
+      lifecycle: existingTemplate.template?.lifecycle,
+      settings: existingTemplate.template?.settings,
+      mappings: existingTemplate.template?.mappings,
+    },
+    version: ASSET_VERSION,
+    _meta: {
+      managed: true,
+      description: `Base settings for the ${definition.name} stream`,
+    },
+  };
+}
+
+export function generateUnwiredLayer(
+  definition: UnwiredStreamDefinition,
+  isServerless: boolean
+): ClusterPutComponentTemplateRequest {
+  return {
+    name: getStreamLayerComponentName(definition.name),
+    template: {
+      lifecycle: getTemplateLifecycle(definition, isServerless),
+      settings: getTemplateSettings(definition, isServerless),
+      mappings: {
+        subobjects: false,
```

Comment:
These mapping changes should not be done - we need to leave the classic stream in their current state as much as possible.

Reply:
removed in https://github.com/elastic/kibana/pull/211368/commits/0ea6334ecafe6839dd86d404e2eddd1fa06b68f1